app/controllers/submissions_controller.rb

```diff
@@ -328,23 +328,23 @@ def view
     @annotations.delete_if do |annotation|
       problem = annotation.problem ? annotation.problem.name : "General"
       if problem != "General" then
-        out = Score.where("submission_id = ? AND  problem_id = ?", @submission.id, Problem.where("assessment_id = ? AND name = ?", @assessment.id, problem).first.id).first.released
-        if out || (@cud.instructor || @cud.course_assistant) then
-          false
-        else
-          if(@assessment.grading_deadline.past?) then 
-            false
-          else
-            true
+        out = Score.find_or_initialize_by_submission_id_and_problem_id(@submission.id, annotation.problem.id)
```

Comment:
How are we checking if the score has been released? Looks like we dropped this functionality.